### PR TITLE
Feature/join population datasets

### PR DESCRIPTION
**Pull request by seb231**
* Load historic datasets
* Create `:state` from `:need` and `:placement`
* Create a row per individual
* Add ids for each individual
* Create a row per simulation and per individual

### CODE REVIEW

**Comment by otfrom** [comment]
using def to hold data you are loading from files off disk is a bad idea as a problem with the file will mean that the repl won't start

**Comment by Eleonore9** [comment]
Thanks @otfrom I changed that!

**Comment by Eleonore9** [comment]
OK, I put `repeat` back instead of `cycle`!

File: src/witan/send/schemas.clj [hunk 1]
```diff
@@ -42,10 +42,11 @@
...
 
 (def SENDSchemaIndividual
-  (make-ordered-ds-schema [[:id s/Int]
-                           [:year (s/constrained s/Int u/year?)]
+  (make-ordered-ds-schema [[:year (s/constrained s/Int u/year?)]
                            [:age (s/constrained s/Int u/SENDage?)]
-                           [:state s/Keyword]]))
+                           [:state s/Keyword]
+                           [:sim-num s/Int]
+                           [:id s/Int]]))
```
Comment (seb231) [inline src/witan/send/schemas.clj]: I'm sure there's a good reason for it, but why was the order changed?

Comment (Eleonore9) [inline src/witan/send/schemas.clj]: @seb231 because we're addind `:sim-num` and then `:id` I think they end up being the two last columns.

**Pull request merged**